index.html

```diff
@@ -118,6 +118,33 @@ <h2>A project for first-time contributions</h2>
       <!-- Keep one line of space above and below your card -->
       <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________  qianqiu card START  ________  -->
+        <div class="card">
+          <p class="name">qianqiu</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">Your handle</a>
```

Comment:
Please replace this placeholder with your twitter profile link if you have an account. If not, feel free to remove this line and the one above.